README.md

```diff
@@ -108,6 +108,14 @@ Then, to run cortex you would start `iex` with the following options:
 CORTEX_ENABLED=true iex -S mix
 ```
 
+## Configuring
+
+```ex
+config :cortex,
+  clear_before_running_tests: true
```

Comment:
On the fence about defaulting this to true since it feels like it might be a kind of jarring change. @russmatney @rschmukler thoughts?

Reply:
I'm mostly indifferent. The readme and changelog document the change well, so, anyone coming from an old version should be able to see what happened easily. Subjectively, I also prefer screen clearing on refresh, so I'd say :shipit: